xmtp/README.md

```diff
@@ -13,12 +13,24 @@
 - Repeated sends of the same payload should be idempotent. When receiving a message or invite, the receiving side will store the hash of the encrypted payload alongside the decrypted result. If a message is received with an id that already exists in the DB, it is ignored.
 - We have ignored race conditions for now (as network requests may take different amounts of time). The receiver side should be tolerant of out-of-order payloads. If ordering is a must, it is possible to use multi-producer, single-consumer queues, or singleton threads for processMessages() and processPayloads() that run on an interval.
 
+### Helper database models
+
+`refresh_jobs`:
+
+```sql
+CREATE TABLE IF NOT EXISTS refresh_jobs (
+    id TEXT PRIMARY KEY NOT NULL, # would be either `invite` or `messages`
+    last_run BIGINT NOT NULL,
+)
+```
+
 ### States
 
 ```
 Conversation:
     - UNINITIALIZED: No invites have been sent
     - INVITED: Invites have been sent
+    - INVITE_RECEIVED: You have been invited to this conversation by another installation
```

Comment:
Technically we could just re-use `INVITED` and everything would be fine. Just don't want to throw out the context as to whether you were the inviter or invitee in a convo. Feels like it might be useful later.